libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -2923,6 +2923,56 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                         VerifyInteger),
                     ReturnType.Array,
                     ValidateBinaryNumber),
+                new ExpressionEvaluator(
```

Comment:
Maybe we can add a number transform method (`NumberTransform`) that convert one number to another. Just like the `StringTransform` method that we have.
This will greatly facilitate us to add number related functions, like `sqrt`, `abs`, `exp`, `log` .etc.

Reply:
NumberTransform added